cmd_actuators_mux/param/actuator_mux.yaml

```diff
@@ -0,0 +1,36 @@
+# Individual subscriber configuration:
+#   name:           Source name
+#   topic:          The topic that provides  messages
+#   timeout:        Time in seconds without incoming messages to consider this topic inactive
+#   priority:       Priority: an UNIQUE unsigned integer from 0 (lowest) to MAX_INT 
+#   short_desc:     Short description (optional)
+
+
+# Fin Angles
+fin_angles_subscribers:
+  - name:        "Autopilot"
+    topic:       "input/autopilot/set_angles"
+    timeout:     0.1
+    priority:    0
+    short_desc:  ""
```

Comment:
@patapufate nit: adding short descriptions would be useful.

Reply:
fixed in 5290206